### ISSUE TITLE
Galils: reinstate positions on TS1 post power cut

### ISSUE THREAD

**Issue by FreddieAkeroyd** [ISISComputingGroup/IBEX#5511]
Request from Jason Chandler

We have a few Galils in TS1 that are not protected by a UPS. They are on Pearl, LOQ, Crisp (x2) and Merlin. The Merlin jaws should be fine as they have pot feedback.

Can you please reinstate the pre power cut positions on each of the above Galils?

Thanks,
Jason

**Comment by John-Holt-Tessella** [comment]
Estimated 3 points but please alter as it becomes clear what we have to do (This is a support ticket).

**Comment by FreddieAkeroyd** [comment]
As we shutdown the control system in March/April, out last recorded positions will be several months old. Scientists contacted to confirm using such positions is OK

**Comment by DominicOram** [comment]
@davidkeymer and @FreddieAkeroyd, given the age of this and the fact we're now in cycle presumably this is done?

**Comment by KathrynBaker** [comment]
@DominicOram I think that is a safe assumption personally, if it isn't done we will get support calls and deal with them that way anyway at this point rather than as part of this ticket

**Comment by davidkeymer** [comment]
I've just sent messages to the respective instrument scientists asking whether their request is still valid and if they need me to do anything and when.  I shall update this ticket when I hear back...

**Comment by davidkeymer** [comment]
New ticket #5732 created to cover remaining work on CRISP.
